README.md

```diff
@@ -153,11 +152,12 @@ cat results.txt >> report.md
 
 :framed_picture: **Images** Display images using the markdown or HTML. Note that
 if an image is an output of your ML workflow (i.e., it is produced by your
-workflow), you will need to use the `cml publish` function to include it a CML
-report. For example, if `graph.png` is output by `python train.py`, run:
+workflow), you can add `--publish` flag to include it a CML report. For example,
+if `graph.png` is output by `python train.py`, run:
 
 ```bash
-cml publish graph.png --md >> report.md
+echo "![](./graph.png)" >> report.md
+cml comment create --publish report.md
```

Comment:
```suggestion
cml comment create report.md
```

Reply:
This example has a local file that needs to be published?

Reply:
Now `--publish=true` by default

Reply:
hmm, I think we should do a follow-up to either have a syntax that can prevent that on a per-link basis or only publish if the file is not tracked by git?

Reply:
Why? 🤔

Reply:
maybe I want to control what gets uploaded and to where? why do we let you self-host your own? why can't you reference an image already in your repo without uploading it somewhere else? I think the reasons for all of these are the same.

Reply:
should I scrub all `--publish` references then?

Reply:
> _maybe I want to control what gets uploaded and to where? why do we let you self-host your own? why can't you reference an image already in your repo without uploading it somewhere else?_

Only files liked from the specified `report.md` will be uploaded. Do you have any use case in mind where users would link to files from `report.md` and not want them to be uploaded? 😅

Reply:
> _should I scrub all `--publish` references then?_

Preferably

Reply:
> Only files liked from the specified `report.md` will be uploaded. Do you have any use case in mind where users would link to files from `report.md` and not want them to be uploaded? 😅

linking any file that's in the repo?

Reply:
E.g. the following code will only upload `/path/to/file.png`

```bash
echo '![example](/path/to/file.png)' > report.md
cml comment create report.md
```

Reply:
Given that users control what they write,there should be no need for more complex solutions

Reply:
I would want more control than the default of uploading every file path in my markdown report, but I'm fine with filing that opinion away if no one else sees the value...